triviagame.html

```diff
@@ -2,48 +2,65 @@
 1. Choose your trivia category: movies, songs, or history.
 2. Answer 4 numerical questions
 3. You will get hints to whether your guess is too high or too low
-4. You have 3 guesses each to get the answer right
-5. (Optional) You have 10 seconds each before your time runs out! -->
+4. !! You have 3 guesses each to get the answer right
+5. !! Add a quit feature to exit the screen at any prompt
+6. !! You have 10 seconds each before your time runs out! 
+7. !! Create objects to manage the trivia and answers. -->
 
 <title>Adam's Trivia Game!</title>
 
 <script>
+
+	var movieQuestions = [
+		"How many films has Christian Bale been in?",
```

Comment:
there's probably a keyboard shortcut that will automagically change them...i hope. yes, please change this one =)